### PR TITLE
build(ios) fixed errors from xcode build

### PR DESCRIPTION
**Pull request by vikas-goyal-axway**
Replaced old configuration with latest, copied same configuration used in socket module.

### CODE REVIEW

File: ios/barcode.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -279,10 +279,7 @@
 				GCC_WARN_UNUSED_PARAMETER = NO;
 				GCC_WARN_UNUSED_VALUE = NO;
 				GCC_WARN_UNUSED_VARIABLE = NO;
-				HEADER_SEARCH_PATHS = (
-					"$(TITANIUM_BASE_SDK)",
-					"$(TITANIUM_BASE_SDK2)",
-				);
+				HEADER_SEARCH_PATHS = "$(HEADER_SEARCH_PATHS)";
```
Comment (janvennemann) [inline ios/barcode.xcodeproj/project.pbxproj]: Please remove this line completely as it does not have any effect.

File: ios/barcode.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -333,10 +330,7 @@
...
```
Comment (janvennemann) [inline ios/barcode.xcodeproj/project.pbxproj]: Same as above

**Comment by janvennemann** [comment]
@sgtcoolguy Any idea why the build keeps failing here on iOS? It doesn't seem to properly launch the app and the just times out. Works fine on my local machine, though.

**Comment by sgtcoolguy** [comment]
@janvennemann It's not really clear to me. It is trying on an Xcode 9.4.1 node. Perhaps it needs Xcode 10/11? I'll try a one-off build on an Xcode-11 node and see if that helps?

**Comment by sgtcoolguy** [comment]
Well, that seemed to be it. I made the change to master branch to use Xcode 11 (and to only publish to NPM on successful master branch builds! Oops, it published a version from his PR)

**Comment by janvennemann** [comment]
@sgtcoolguy Ah yeah, iirc the Titanium Karma launcher needs Xcode 10+ for Swift 5. Thanks!

**Pull request merged**